### ISSUE TITLE
Aur 404

### ISSUE THREAD

**Issue by Tealk** [topgrade-rs/topgrade#103]
Hi, in the AUR I get the following error message:
```
url: (22) The requested URL returned error: 404
==> ERROR: Failure while downloading https://github.com/topgrade-rs/topgrade/archive/v10.1.0.tar.gz
```

**Comment by musicfunfan** [comment]
Yea i get the same error. 
I think is problem of wrong link... or something like that...

**Comment by yuhldr** [comment]
github released 10.1.0 has been deleted, so ....

**Comment by DottoDev** [comment]
I know, because 10.1.0 currently fails the CD Pipeline because of some problems with the clap dependency
